cmd/oci-create-runtime-bundle/main.go

```diff
@@ -29,6 +30,9 @@ import (
 // and will be populated by the Makefile
 var gitCommit = ""
 
+// refRegexp for string ref should be complied with description "Object names in the refs subdirectories MUST NOT include characters outside of the set of "A" to "Z", "a" to "z", "0" to "9", the hyphen -, the dot ., and the underscore _." at https://github.com/opencontainers/image-spec/blob/master/image-layout.md.
+var refRegexp = regexp.MustCompile(`^[a-zA-Z0-9\-\.\_]+?$`)
```

Comment:
This constant should probably be provided by image-spec, from which runtime-tools can import and use it.